pkg/controller/statusmanager/status_manager.go

```diff
@@ -10,6 +10,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/openshift/cluster-network-operator/pkg/names"
```

Comment:
Wrong repo, should be kubevirt/cluster-network-addons-operator

Reply:
Right, fixed, thanks